src/Clock.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import "./Clock.css";
+
+export default function Clock(props) {
+  const [date, refreshDate] = React.useState(new Date());
+
+  function refreshClock() {
+    refreshDate(new Date());
+  }
+
+  React.useEffect(() => {
+    const timerId = setInterval(refreshClock, 1000);
+    return () => {
+      clearInterval(timerId);
+    };
+  }, []);
+
+  return (
+    <div>
+      <table>
+        <tbody>
+          <tr>
+            <td>{props.timezone.split("/")[1]}</td>
```

Comment:
While this definitely works, just a general reminder that this might lead to bugs. The props are not validated and one could possibly pass an integer as time zone as well. If you want to safeguard this a bit, you could look into React PropTypes